IdentityCore/src/oauth2/token/MSIDBaseToken.m

```diff
@@ -64,7 +64,7 @@ - (BOOL)isEqual:(id)object
 
 - (NSUInteger)hash
 {
-    NSUInteger hash = [super hash];
+    NSUInteger hash = 0;
```

Comment:
I think this is already fixed in dev?

Reply:
yes, it is already in dev. I merged that branch into this one as well.